src/components/Person/_person.scss

```diff
@@ -105,17 +111,35 @@
   }
 }
 
-.cc-person--author .cc-person__image + .cc-person__wrapper {
-  @include mq(sm) {
-    flex-basis: calc(100% - var(--person-image-size-small));
-  }
-}
-
 .cc-person--author .cc-person__heading {
   font-size: var(--heading-xs);
 }
 
 .cc-person--author .cc-person__description,
-.cc-person--author a {
+.cc-person--author .cc-person__link {
   font-size: var(--body-sm);
 }
+
+// variant: NARROW
+
+.cc-person--narrow .cc-person__image {
+  @include mq(md) {
+    position: absolute;
+  }
+
+  @include mq(xl) {
+    position: static;
+  }
```

Comment:
To avoid having to declare `position` twice, you could

```scss
@include mq($from: md, $until: xl) {
  position: absolute;
}
```

https://github.com/sass-mq/sass-mq#responsive-mode-on-default